lib/internal/process/per_thread.js

```diff
@@ -146,8 +146,10 @@ function wrapProcessMethods(binding) {
   }
 
   const memValues = new Float64Array(5);
-  function memoryUsage() {
-    _memoryUsage(memValues);
+  function memoryUsage(options = {}) {
+    const rssOnly = options.rssOnly || false;
```

Comment:
```suggestion
    const rssOnly = !!(options && options.rssOnly);
```
Guards against `null` and `undefined` and always produces a boolean.